test/spec/dataviews/histogram-dataview-model.spec.js

```diff
@@ -340,4 +340,70 @@ describe('dataviews/histogram-dataview-model', function () {
       expect(this.model.get('own_filter')).toBeUndefined();
     });
   });
+
+  describe('._sumBinsFreq', function () {
```

Comment:
I'm a big fan of testing everything as easy as possible and having private methods available eases a lot this process.

In this particular case, a couple of tests in `parse` can check all the paths, so I got rid of those private methods specs and added a couple testing `parse`.

"We may have lost the battle, but not the war"